### DISCUSSION TITLE
Debug is greyed out, system is not recognized

### DISCUSSION THREAD

**Discussion by alexlvcom** [devcompl/TTController#169] [Q&A] [answered]
**Describe the issue**
Run in Console and Debug commands are greyed out. I can only Manage service, install and run it. 

**My hardware**
AsRock X570 Steel Legen motherboard
Thermaltake Sync Controller TT Premium Edition connected to motherboard's ADDR LED pin socket. 
Riing Plus 20 and Riing Plus 12 fans connected to controller.
Fan leds can be set up with mainboard bios tool successfully, I can change effects and everything.
TT RGB Plus sotware doesn't work, giving h_0x0001 error, so I tried  MoshiMoshi0/TTController software, which also doesn't work.

**Comment by MoshiMoshi0** [comment, marked as answer]
Sync controller can only be controlled by the motherboard software, it is not visible to TT RGB Plus or TTController because there is no usb connection.
For speed control I think there is a fan input that you connect to the motherboard.

> Run in Console and Debug commands are greyed out

The service must be stopped for them to not be greyed out.

**Reply by alexlvcom** [reply]
Thanks for quick answer. I stopped the service and now Debug option appeared, and it doesn't list any controllers, that confirms your words about no usb connection and it makes sense.

**Comment by CrEye1** [comment]
.What a drag!  I'm in the same boat, I bought 15x Riing Trios on FB marketplace, bought the premium Sync Controller and would love more control.

So the controller has 3 channels, 3 fans in each channel, and a PWM control port for each channel.

I put 2x Riing Trios (TT-1225) in each channel, and only connected one of the PWM ports to the mobo.  Are we stuck with this limited control?  All 6 fans essentially show up as one (thanks to the TT premium controller)

**Reply by CrEye1** [reply]
I thought I would have control of 3 separate pairs with this configuration.

**Reply by MoshiMoshi0** [reply]
Sync controller is very limited and depends on motherboard software for rgb/fan speed. You need to get Riing Trio or Riing Quad controller.

**Reply by CrEye1** [reply]
> Sync controller is very limited and depends on motherboard software for rgb/fan speed. You need to get Riing Trio or Riing Quad controller.

This is looking quite tough to find a "controller only" listing.  If I wanted an original controller, you have any ideas where to get lucky?
